Resync web-platform-tests/dom from upstream

cdumez opened the pull request:
#### ba7577b539ac1459e95f5b21973010f4e43b5e94
<pre>
Resync web-platform-tests/dom from upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=243176">https://bugs.webkit.org/show_bug.cgi?id=243176</a>

Reviewed by Tim Nguyen.

Resync web-platform-tests/dom from upstream 6c4231c465fc3ed7b928.

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/document-level-touchmove-event-listener-passive-by-default.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/document-level-touchmove-event-listener-passive-by-default.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/document-level-wheel-event-listener-passive-by-default.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/document-level-wheel-event-listener-passive-by-default.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/resources/scrolling.js: Added.
(runTest):
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/resources/touching.js: Added.
(waitForCompositorCommit):
(runTest):
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/resources/wait-for.js: Added.
(waitFor):
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/synthetic-events-cancelable-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/synthetic-events-cancelable.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/252824@main">https://commits.webkit.org/252824@main</a>
</pre>

nt1m reviewed (approved):
r=me if EWS is happy

cdumez committed:
Resync web-platform-tests/dom from upstream
https://bugs.webkit.org/show_bug.cgi?id=243176

Reviewed by Tim Nguyen.

Resync web-platform-tests/dom from upstream 6c4231c465fc3ed7b928.

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/dom/events/document-level-touchmove-event-listener-passive-by-default.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/document-level-touchmove-event-listener-passive-by-default.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/document-level-wheel-event-listener-passive-by-default.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/document-level-wheel-event-listener-passive-by-default.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-mousewheel-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchmove-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-touchstart-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/non-passive-wheel-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-mousewheel-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchmove-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-touchstart-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-body-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-div-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-div.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-root-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-window-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/passive-wheel-event-listener-on-window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/resources/scrolling.js: Added.
(runTest):
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/resources/touching.js: Added.
(waitForCompositorCommit):
(runTest):
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/resources/wait-for.js: Added.
(waitFor):
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/synthetic-events-cancelable-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/synthetic-events-cancelable.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/non-cancelable-when-passive/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/dom/events/w3c-import.log:

Canonical link: https://commits.webkit.org/252824@main
